.github/workflows/create-deb.yaml

```diff
@@ -0,0 +1,34 @@
+name: create-deb
+
+on:
+  release:
+    types: [released]
+  pull_request:
+    paths:
+      - .github/workflows/create-deb.yaml
+  workflow_call:
+
+jobs:
+  package:
```

Comment:
```suggestion
  package:
  # This doesn't currently publish anywhere. We can add publishing in a follow-up PR
```

Reply:
Yeah, I don't really know what happens with the build, maybe it ends up somewhere where you can add it as an artifact on a release or something? Not sure how this works.

Reply:
Yes, we can do something like https://github.com/PRQL/prql/blob/4a8b915eef940b5ef1231b915ceaa210b7a1e86b/.github/actions/time-compilation/action.yaml#L29 for normal artifacts.

Then there are also "release artifacts", but that requires running the whole release through GitHub, which we don't do yet https://github.com/softprops/action-gh-release